packages/plugins/typescript/src/visitor.ts

```diff
@@ -81,15 +81,15 @@ export class TsVisitor extends BaseTypesVisitor<TypeScriptPluginConfig, TypeScri
       return new DeclarationBlock(this._declarationBlockConfig)
         .export()
         .asKind('type')
-        .withName(this.convertName(node.name))
+        .withName(this.convertName(node))
```

Comment:
Please make sure that we don't need to make similar changes for `flow` and all other plugins